docs/src/bectl/build.md

```diff
@@ -0,0 +1,26 @@
+# Build & Test
+
+
+Building `bectl` is relatively simple but does require all dependencies of
+`betree_storage_stack` to be available. If you have not done this yet refer to
+the [Building chapter](../build.md).
+
+Given all prerequisites are fulfilled simply run
+
+```sh
+$ cargo build
+```
+
+from the `bectl` directory to build bectl on its own.
+
+To avoid path specification on each invocation of the application you may also install the app via `cargo` to your local user.
```

Comment:
This was meant to say that to avoid calling the `bectl` binary on each invocation with `../target/debug/bectl` or with `cargo run` from the directory the user may just install it in his home `.cargo` directory. Then "normal" invocation with `bectl` in the shell is possible.